### PR TITLE
[MARLIN-438] supporting creditcard account_type

### PR DESCRIPTION
**Pull request by hhemanth**
Added logic to identify creditcard account and suitably populate acct_type field

### CODE REVIEW

File: lib/ofx/parser/ofx220.rb [hunk 1]
```diff
@@ -16,9 +16,14 @@ def build_account_info(node)
           description:       node.search('desc').inner_text.upcase,
           bank_id:           nested_account_info.search('bankid').inner_text,
           id:                nested_account_info.search('acctid').inner_text,
-          type:              ACCOUNT_TYPES[node.search('accttype').inner_text.to_s.upcase],
+          type:              fetch_account_type(node),
         })
       end
+
+      def fetch_account_type(node)
```
Comment (ouranos) [inline lib/ofx/parser/ofx220.rb:23]: I don't think you need this method here as it's inheriting from `ofx102`

**Review by ouranos — CHANGES_REQUESTED** [review]
`staging` is an integration branch so we shouldn't branch of it.

Look at how the other branches are structured:
- branched off of `master`
- PR open against the upstream repo
- Pulled into `staging` while waiting for the PR to be merged

This case is a bit specific because it's related to the `account_discovery` branch where we haven't open a PR against the main repo yet.
I'd suggest you branch off of `account_discovery` and I'll push you commit to the branch.
Also since we're likely to push it to the main repo, you can drop the jira issue id as that's internal

File: lib/ofx/parser/ofx220.rb (hunk 1 above)
Comment (hhemanth) [reply lib/ofx/parser/ofx220.rb:23]: Both the methods are different especially the line below
ACCOUNT_TYPES[node.search("bankacctfrom > accttype").inner_text.to_s.upcase]
ACCOUNT_TYPES[node.search('accttype').inner_text.to_s.upcase]

**Comment by ouranos** [comment]
Superseded by  #3

**Pull request closed without merging**